src/azure-cli/azure/cli/command_modules/acr/task.py

```diff
@@ -72,11 +73,34 @@ def acr_task_create(cmd,  # pylint: disable=too-many-locals
                     resource_group_name=None,
                     assign_identity=None,
                     target=None,
-                    auth_mode=None):
+                    auth_mode=None,
+                    log_template=None,
+                    is_system_task=False):
 
     registry, resource_group_name = get_registry_by_name(
         cmd.cli_ctx, registry_name, resource_group_name)
 
+    Task = cmd.get_models('Task')
+
+    # If quicktask skip other parameters
+    if is_system_task and task_name == ACR_TASK_QUICKTASK:
+        task_create_parameters = Task(
+            location=registry.location,
+            status=status,
+            timeout=timeout,
+            log_template=log_template,
+            is_system_task=is_system_task)
+        try:
+            return client.create(resource_group_name=resource_group_name,
+                                 registry_name=registry_name,
+                                 task_name=task_name,
+                                 task_create_parameters=task_create_parameters)
+        except ValidationError as e:
+            raise CLIError(e)
+
+    if not context_path:
+        raise CLIError("If the task is not a System Task, --context-path must be provided.")
```

Comment:
can this message be put to help message?

Reply:
@yungezz updated the help section.